after-install.sh

```diff
@@ -17,7 +17,7 @@ create_user() {
   FILES="${FILES} /usr/local/share/lua/"
 
   for FILE in ${FILES}; do
-    chown -R kong:kong ${FILE}
+    chown -R kong:root ${FILE}
     chmod -R g=u ${FILE}
```

Comment:
Given that the group permissions are set to mirror the user permissions, it's conceivable that something is using the group perms to access the file contents.  So changing the group ownership from `kong` to `root` would break those things.

More importantly, this should probably be root:kong, followed by a chmod which removes group write.  That way, anything accessing as group kong can still read and execute as before, but can no longer write.  The kong user should be a member of the kong group, so anything formerly running as `kong` will still be able to read and execute by virtue of being in the kong group.
```suggestion
    chown -R root:kong ${FILE}
    chmod -R g=u,g-w ${FILE}
```